tests/unit_tests/test_deplete_activation.py

```diff
@@ -98,7 +111,7 @@ def test_activation(run_in_tmpdir, model):
     _, atoms = results.get_atoms(str(w.id), "W186")
 
     assert atoms[0] == pytest.approx(n0)
-    assert atoms[1] / atoms[0] == pytest.approx(0.5, rel=1e-3)
+    assert atoms[1] / atoms[0] == pytest.approx(0.5, rel=0.01)
```

Comment:
Perhaps we could include a tolerance with the test parameters to maintain a better check when using the direct reaction rates to do depletion.

Reply:
Good call. Looks like I can crank down the tolerance when using direct reaction rates to 1e-5 or even lower.